documentation/book/cluster-operator.adoc

```diff
@@ -515,6 +518,27 @@ Furthermore, containers doing lots of disk I/O (such as Kafka broker containers)
 for use as operating system page cache. On such containers, the request memory should be substantially more than the
 memory used by the JVM.
 
+[[kafka_rack]]
+===== Kafka rack
+
+It is possible to enable Kafka rack-awareness by specifying a JSON string for the `kafka-rack` field in the cluster ConfigMap.
```

Comment:
_specifying JSON string_ sounds really strange. It should probably be something like _specifying JSON object as a string_

Reply:
Let's use just "JSON object" as already done in other parts in the doc. Maybe saying "as a string" is redundant.